docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Make sure it 100% corresponds to: https://docs.minaprotocol.com/zkapps/writing-a-zkapp/introduction-to-zkapps/how-to-deploy-a-zkapp
(including content itself + correct belonging to the docs structure)

Reply:
was duplicated - I've removed the extra one